ChemistryLib/PhreeqcIO.cpp

```diff
@@ -80,6 +92,27 @@ PhreeqcIO::PhreeqcIO(std::string const project_file_name,
             "will write output.",
             _output->basic_output_setups.output_file.c_str());
     }
+
+    if (_dump)
+    {
+        SetDumpFileOn(phreeqc_instance_id, 1);
```

Comment:
comment on the magic number would be nice

Reply:
Added.